modules/fluentd/README.md

```diff
@@ -0,0 +1,129 @@
+required to retrieve credentials to write to S3.
+
+You must enable and configure the
+[AWS Secrets Engine](https://www.vaultproject.io/docs/secrets/aws/index.html) to allow the Vault
+and the job to retrieve credentials. This module will provision the appropriate Vault AWS Secrets
+Engine role and the IAM policies required.
+
+Provide the necessary paths to the module variables.
+
+## Default settings
+
+By default, `fluentd` is configured to match tags from logs sent from the example configuration
+provided in the [`td-agent`](../td-agent) module. See the module for more information on how to
+configure your instances to forward logs to fluentd. It also match logs tagged with `docker.*` for
+your Nomad jobs.
+
+You can change the matched tags with the `fluentd_match` variable.
+
+## Applying the module
+
+There are some things to take note of before applying the module other than the requirements above.
+
+### Fluentd port
+
+Fluentd will statically bind itself to a port of your choose via the `fluentd_port` variable on your
+Nomad clients.
+
+In order for your applications to forward logs to your Fluentd servers, you will have to define
+additional security group rules to your Nomad clients cluster.
+
+### Elasticsearch Address and Port
+
+If you provisioned Elasticsearch with the [elasticsearch module](../elasticsearch) module, you can
+provide the address to Elasticsearch using the Consul service catalog.
+
+For example:
+
+```hcl
+data "consul_catalog_service" "elasticsearch" {
+  name = "elasticsearch"
+}
+
+module "fluentd" {
+  # ...
+  elasticsearch_hostname = "${data.consul_catalog_service.elasticsearch.service.0.address}"
+  elasticsearch_port     = "${data.consul_catalog_service.elasticsearch.service.0.port}"
+}
+
+```
+
+## Forwarding Logs
+
+You can use the [td-agent module](../td-agent) along with the example configuration files to forward
+logs from your Consul Servers, Noamd Servers, Nomad Clients, and Vault Servers to Fluentd.
+
+If you would like to forward logs from your Nomad jobs, you might want to tag them with
+`docker.XXX`.
+
+For example, in your Jobspec, you can use:
+
+```hcl
+job "job" {
+  # ...
+  group "group" {
+    # ...
+    task "task" {
+      # ...
+      driver = "docker"
+
+      config {
+        logging {
+          type = "fluentd"
+
+          config {
+            fluentd-address = "fluentd.service.consul:4224"
+            tag             = "docker.job"
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| aws_region | Region of AWS for which this is deployed | string | `ap-southeast-1` | no |
+| consul_key_prefix | Path prefix to the key in Consul to set for the `core` module to know that this module has         been applied. If you change this, you have to update the         `integration_consul_prefix` variable in the core module as well. | string | `terraform/` | no |
```

Comment:
The whitespaces looks weird here, maybe you want to try re-doing this `terraform-docs` part.

Reply:
No difference. It doens't render out though. Easier to "maintain" by just copy/pasting.